packages/react-notifications/src/components/notification-kinds/api-error-message/messages.ts

```diff
@@ -21,7 +21,7 @@ export default defineMessages({
     description:
       'User does a bulk update but someone else has saved changes for that element',
     defaultMessage:
-      'Sorry, but we were unable to save your changes as someone else made changes to this same resource while you were editing.',
+      'Sorry, but we were unable to save your changes as someone else made changes to this same source while you were editing.',
```

Comment:
These changes simply reflect the values in the `core.json`, so that by running the extract command we don't overwrite/change the values.